magento/magento2
# Check if copy method is copy before sending comment email

### Description (*)
As seen in the comments email copies should only be sent as separated emails if their copy method is copy.

### Related Pull Requests
None I could find.

### Fixed Issues (if relevant)
None I could find.

### Manual testing scenarios (*)
1. Enable `sales_email/order_comment/enabled`
2. Add email address to `sales_email/order_comment/copy_to`
3. Set `sales_email/order_comment/copy_method` to bcc (default value).
4. Place order.
5. Add order comment in order overview (adminhtml area) and uncheck notify customer.

6. Email copy should not be send because notify customer is disabled and copy set to bcc.

### Questions or comments

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29915: Check if copy method is copy before sending comment email
